packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -426,6 +426,15 @@ const double _kDefaultFontSize = 14.0;
 ///
 /// ### Inconsistent platform fonts
 ///
+/// By default, fonts differ depending on the platform.
+///
+///  * The default font-family for `Android`,`Fuchsia` and `Linux` is `Roboto`.
+///  * The default font-family for `iOS` is `.SF UI Display`/`.SF UI Text`.
+///  * The default font-family for `MacOS` is `.AppleSystemUIFont`.
+///  * The default font-family for `Windows` is `Segoe UI`.
+//
+// The implementation of these defaults can be found in /packages/flutter/lib/src/material/typography.dart
```

Comment:
Sorry to be picky, but can you break this up into two lines:

```dart
// The implementation of these defaults can be found in:
// /packages/flutter/lib/src/material/typography.dart
```

As that makes it a little easier to read on narrow displays.

Reply:
@darrenaustin done.